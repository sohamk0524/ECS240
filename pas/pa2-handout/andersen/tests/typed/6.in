c Test case with mix of all statements
p 55 80
t 1 10
t 2 10
t 3 10
t 4 10
t 5 10
t 6 9
t 7 9
t 8 9
t 9 9
t 10 9
t 11 8
t 12 8
t 13 8
t 14 8
t 15 8
t 16 7
t 17 7
t 18 7
t 19 7
t 20 7
t 21 6
t 22 6
t 23 6
t 24 6
t 25 6
t 26 5
t 27 5
t 28 5
t 29 5
t 30 5
t 31 4
t 32 4
t 33 4
t 34 4
t 35 4
t 36 3
t 37 3
t 38 3
t 39 3
t 40 3
t 41 2
t 42 2
t 43 2
t 44 2
t 45 2
t 46 1
t 47 1
t 48 1
t 49 1
t 50 1
t 51 0
t 52 0
t 53 0
t 54 0
t 55 0
s 0 1 -1 10
s 0 5 -1 9
s 0 3 -1 7
s 0 1 0 1
s 0 5 0 5
s 0 3 0 1
s 0 1 0 5
s 0 5 0 1
s 0 6 -1 11
s 0 10 -1 14
s 0 8 -1 12
s 1 5 1 1
s 1 3 1 5
s 0 8 0 8
s 0 8 0 6
s 0 6 1 3
s 0 11 -1 19
s 0 15 -1 16
s 0 13 -1 19
s 2 1 0 15
s 0 13 1 8
s 1 10 2 1
s 1 10 1 10
s 1 6 2 3
s 0 16 -1 24
s 0 20 -1 24
s 0 18 -1 21
s 0 20 2 6
s 3 5 1 15
s 2 6 0 18
s 3 1 2 8
s 0 20 2 8
s 0 21 -1 26
s 0 25 -1 27
s 0 24 -1 26
s 0 21 0 25
s 4 5 3 6
s 0 25 4 3
s 2 13 1 20
s 4 3 3 8
s 0 26 -1 35
s 0 30 -1 33
s 0 27 -1 35
s 1 25 1 25
s 0 27 0 26
s 4 10 1 25
s 0 30 1 21
s 0 26 4 6
s 0 31 -1 36
s 0 35 -1 37
s 0 32 -1 37
s 6 1 2 24
s 3 20 4 11
s 6 5 2 25
s 4 13 2 25
s 1 26 5 8
s 0 36 -1 42
s 0 40 -1 42
s 0 37 -1 45
s 5 13 6 8
s 3 25 0 40
s 1 32 3 25
s 1 35 1 35
s 5 11 2 30
s 0 41 -1 46
s 0 45 -1 48
s 0 42 -1 47
s 2 31 7 8
s 6 13 1 40
s 0 41 7 6
s 0 41 6 11
s 3 30 1 36
s 0 46 -1 54
s 0 50 -1 53
s 0 49 -1 51
s 4 30 8 10
s 3 32 9 1
s 9 1 3 31
s 3 31 4 27
s 3 35 0 46
