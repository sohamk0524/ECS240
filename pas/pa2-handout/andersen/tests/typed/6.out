pt 10 14
pt 1 10
pt 11 16
pt 11 19
pt 13 16
pt 13 19
pt 14 16
pt 15 16
pt 16 21
pt 16 24
pt 18 21
pt 1 9
pt 19 21
pt 20 21
pt 20 24
pt 21 26
pt 21 27
pt 24 26
pt 24 27
pt 25 26
pt 25 27
pt 26 35
pt 27 35
pt 30 33
pt 30 35
pt 3 10
pt 31 36
pt 32 37
pt 35 37
pt 36 42
pt 3 7
pt 37 42
pt 37 45
pt 3 9
pt 40 42
pt 41 46
pt 41 47
pt 41 48
pt 42 47
pt 42 48
pt 45 47
pt 45 48
pt 46 54
pt 47 54
pt 48 54
pt 49 51
pt 50 53
pt 5 10
pt 5 9
pt 6 11
pt 6 14
pt 7 14
pt 8 11
pt 8 12
pt 8 14
pt 9 14
